src/client/metadataApiRetrieve.ts

```diff
@@ -217,9 +221,13 @@ export class MetadataApiRetrieve extends MetadataTransfer<MetadataApiRetrieveSta
       } as ScopedPreRetrieve);
     }
 
+    const manifestData = (await this.components.getObject()).Package;
+
     const requestBody: RetrieveRequest = {
-      apiVersion: this.components.apiVersion ?? (await connection.retrieveMaxApiVersion()),
-      unpackaged: (await this.components.getObject()).Package,
+      // This apiVersion is only used when the version in the package.xml (manifestData) is not defined.
+      // see docs here: https://developer.salesforce.com/docs/atlas.en-us.api_meta.meta/api_meta/meta_retrieve_request.htm
```

Comment:
excellent comment

Reply:
the `apiVersion` on RetrieveRequest necessary?  We've got code in `getObject()` that is guaranteed to put an apiVersion on the package.xml (looking at the coverage report if necessary).

This path might not ever be hit

Reply:
I think it's required by the metadata API if the docs are correct: https://developer.salesforce.com/docs/atlas.en-us.api_meta.meta/api_meta/meta_retrieve_request.htm

Reply:
ahh, it's required but ignored by the mdapi.  I see.